OutOfSchool/OutOfSchool.WebApi/Extensions/MappingExtensions.cs

```diff
@@ -179,6 +179,15 @@ public static WorkshopCard ToModelSimple(this Workshop workshop)
             });
         }
 
+        public static CompanyInformationDto ToModel(this CompanyInformation companyInformation)
```

Comment:
Please don't introduce these converters on project level as they might be used not only for testing purposes. 
An issue with such implementation is performance penalty for case of processing big amount of entities, that's why we prepare and register mapper once in DI and not recreate it for each processed DTO/model.

So either move and isolate this code in tests or rewrite tests (if possible, ofc)

Reply:
Curently we're trying to switch to using mapping profile and inject the mapper instead of static extensions.
Mapping profile is here: [https://github.com/ita-social-projects/OoS-Backend/blob/develop/OutOfSchool/OutOfSchool.WebApi/Util/MappingProfile.cs](https://github.com/ita-social-projects/OoS-Backend/blob/develop/OutOfSchool/OutOfSchool.WebApi/Util/MappingProfile.cs)
Example usage: [https://github.com/ita-social-projects/OoS-Backend/blob/develop/OutOfSchool/OutOfSchool.WebApi/Services/ClassService.cs#L51](https://github.com/ita-social-projects/OoS-Backend/blob/develop/OutOfSchool/OutOfSchool.WebApi/Services/ClassService.cs#L51)